content/zh-cn/docs/reference/kubernetes-api/cluster-resources/runtime-class-v1.md

```diff
@@ -0,0 +1,588 @@
+-->
+RuntimeClass 定义集群中支持的容器运行时类。
+RuntimeClass 用于确定哪个容器运行时用于运行某 Pod 中的所有容器。
+RuntimeClass 由用户或集群制备程序手动定义，并在 PodSpec 中引用。
+Kubelet 负责在运行 Pod 之前解析 RuntimeClassName 引用。
+有关更多详细信息，请参阅
+https://kubernetes.io/zh-cn/docs/concepts/containers/runtime-class/
+
+<hr>
+
+- **apiVersion**: node.k8s.io/v1
+
+- **kind**: RuntimeClass
+
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **handler** (string), required
+  Handler specifies the underlying runtime and configuration that the CRI implementation will use to handle pods of this class. The possible values are specific to the node & CRI configuration.  It is assumed that all handlers are available on every node, and handlers of the same name are equivalent on every node. For example, a handler called "runc" might specify that the runc OCI runtime (using native Linux containers) will be used to run the containers in a pod. The Handler must be lowercase, conform to the DNS Label (RFC 1123) requirements, and is immutable.
+-->
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  更多信息：
+  https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **handler** (string)，必需
+
+  handler 指定底层运行时和配置，在 CRI 实现过程中将使用这些运行时和配置来处理这个类的 Pod。
+  可能的值特定于节点和 CRI 配置。
+  假定所有 handler 可用于每个节点上，且同一名称的 handler 在所有节点上是等效的。
+  例如，一个名为 “runc” 的 handler 可能指定 runc OCI 运行时将（使用原生 Linux 容器）
+  用于运行 Pod 中的容器。该 handler 必须采用小写，遵从 DNS Label (RFC 1123) 要求，且是不可变更的。
+
+<!--
+- **overhead** (Overhead)
+  Overhead represents the resource overhead associated with running a pod for a given RuntimeClass. For more details, see
+   https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/
+
+  <a name="Overhead"></a>
+  *Overhead structure represents the resource overhead associated with running a pod.*
+-->
+- **overhead** (Overhead)
+
+  overhead 表示运行给定 RuntimeClass 的 Pod 时所关联的资源开销。有关更多详细信息，请参阅
+  https://kubernetes.io/zh-cn/docs/concepts/scheduling-eviction/pod-overhead/
+
+  <a name="Overhead"></a>
+  **Overhead 结构表示运行一个 Pod 所关联的资源开销。**
+  
+  <!--
+  - **overhead.podFixed** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+    
+    PodFixed represents the fixed resource overhead associated with running a pod.
+  -->
+  - **overhead.podFixed** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
```

Comment:
After several hours experimentation and docs digging, I believe I've found a resolution for the indentation and code style issue. For all list items like line 84, we need to **preserve an empty line before them**.

So one empty line before line 84, one empty line before line 107, etc, will trigger the special handling in GoldMark. It will know they are list items and it will render the following paragraphs correctly.

Reply:
ok, thanks. Let me have a try too. Happy weekend :)

Reply:
I refer to this **preserve an empty line before them**, but GoldMark cannot know they are list items